manifests/site.pp

```diff
@@ -111,3 +111,14 @@
 node 'bounce' {
   include role::bounce
 }
+
+## New VMs (Azure) for trusted.ci.jenkins.io
```

Comment:
```suggestion
# New VMs (Azure) for trusted.ci.jenkins.io
```

Reply:
The above "bounce" node L110-113 will be removed after? If so, I would add a TODO comment.

Reply:
> The above "bounce" node L110-113 will be removed after? If so, I would add a TODO comment.

Good idea, even though it should not be part of this PR because we do not know, in the current state, if the former bounce directive will have to be removed or not (what if we rollabck the  change?).

It's the responsibility of the parent issue to track this. "TODO comment" are not a guarantuee that it will be treated

Reply:
Maybe not a TODO but at least a comment on either of them explaining why there are two trusted "bounces"? It feels weird to me, it will certainly feel weirder later ^^

Reply:
TODO added with an explanation that **should** content both point of view: is that ok for you?